### PR TITLE
Grabbag

### PR DESCRIPTION
**Pull request by jeffling**
did some changes without modifying the underlying structure

- removed fusion, not a thing
- remove recipes
- removed graphql sink, not a real thing
- updated copy
- removed the closed beta banner from a bunch of sinks, made no sense

### CODE REVIEW

File: src/components/Layout.jsx [hunk 1]
```diff
@@ -24,22 +24,17 @@ const navigation = [
...
-      { title: "NFT transfers", href: "/recipes/nft-transfers" },
-    ],
-  },
+  // {
+  //   title: "Recipes",
+  //   links: [
+  //     { title: "Overview", href: "/recipes" },
+  //     { title: "NFT transfers", href: "/recipes/nft-transfers" },
+  //   ],
+  // },
```
Comment (patsissons) [inline src/components/Layout.jsx:33]: leaving this commented out?

File: src/components/Layout.jsx [hunk 2]
```diff
@@ -58,10 +58,13 @@ const navigation = [
         title: "What are mirror pipelines?",
         href: "/mirror/what-are-mirror-pipelines",
       },
-      { title: "Quick Start", href: "/mirror/quick-start" },
-      { title: "Creating a pipeline", href: "/mirror/create-a-pipeline" },
       {
-        title: "Choose the right sink",
+        title: "Guide: Export Bored Ape Contract Events",
+        href: "/mirror/guide-bored-ape",
```
Comment (patsissons) [inline src/components/Layout.jsx:63]: maybe a 302 redirect for this one?

File: src/pages/subgraphs/dedicated-subgraphs.md [hunk 3]
```diff
@@ -1,18 +1,20 @@
 ---
-title: Dedicated Subgraphs
+title: Dedicated Subgraph Indexing
```
Comment (patsissons) [inline src/pages/subgraphs/dedicated-subgraphs.md:2]: change the layout file too? route change?

File: src/pages/subgraphs/dedicated-subgraphs.md [hunk 4]
```diff
@@ -1,18 +1,20 @@
...
 description: Index your subgraphs - even on private chains - on dedicated infrastructure for improved customizability and performance.
 ---
 
-The quickest way to get started with Goldsky Subgraphs is with [Shared Subgraphs](/subgraphs/shared-subgraphs). When you need improved customizability and performance, Goldsky offers Dedicated Subgraphs. As the name suggests, this is a dedicated environment for you and you only, allowing for customization and optimization at both the indexing and querying layers.
+The quickest way to get started with Goldsky Indexing is with [Shared Subgraph Indexing](/indexing/shared-subgraph-indexing).
```
Comment (patsissons) [inline src/pages/subgraphs/dedicated-subgraphs.md:6]: this is the current route, up to you if you want change the routes

```suggestion
The quickest way to get started with Goldsky Indexing is with [Shared Subgraph Indexing](/indexing/shared-subgraphs).
```

File: src/pages/subgraphs/shared-subgraphs.md [hunk 5]
```diff
@@ -1,8 +1,13 @@
...
-This is the default subgraphs infrastructure Goldsky provides. It's a fully auto-scaled environment shared among anyone who uses shared subgraphs.
+By default, all subgraphs that are created in Goldsky go into our highly optimized infrastructure.
 
-If you prefer a completely isolated, dedicated infrastructure, please review [Dedicated Subgraphs](/subgraphs/dedicated-subgraphs).
+The platform is fully autoscaling, with a re-engineered RPC and storage layer, and is tuned for fast indexing across the majority of use-cases. It's also completely backwards compatible and runs the same WASM engine as the vanilla open-source graph-node engine.
+
+- Optimized RPC multi-provider layer with a global cache that uses a combination of dedicated and commercial RPC APIs for uptime
+- I/O optimized database with under 1ms average commit times
+
+If you prefer a completely isolated and dedicated infrastructure, please review [Dedicated Subgraph Indexing](/indexing/dedicated-subgraph-indexing).
```
Comment (patsissons) [inline src/pages/subgraphs/shared-subgraphs.md:13]: similar

```suggestion
If you prefer a completely isolated and dedicated infrastructure, please review [Dedicated Subgraph Indexing](/indexing/dedicated-subgraphs).
```

**Review by patsissons — APPROVED** [review]

File: src/components/Layout.jsx (hunk 1 above)
Comment (jeffling) [reply src/components/Layout.jsx:33]: yes, will be adding something back for guides

File: src/pages/subgraphs/dedicated-subgraphs.md (hunk 3 above)
Comment (jeffling) [reply src/pages/subgraphs/dedicated-subgraphs.md:2]: Got a follow up PR for this coming! good callout

Comment (jeffling) [reply src/pages/subgraphs/dedicated-subgraphs.md:2]: fixed it now

**Pull request merged**